Packs/MISP/Integrations/MISPV3/MISPV3_test.py

```diff
@@ -621,3 +685,120 @@ def test_warninglist_response(mocker):
+                                         {'id': '290724', 'event_id': '21342', 'object_id': '39958', 'object_relation': 'from',
+                                          'category': 'Payload delivery', 'type': 'email-src', 'value1':
+                                          'test sender <fake_mail_sender@test.com>', 'value2': '', 'to_ids': True, 'uuid':
+                                          '568f7d28-9e14-4c62-8a22-84084240e02f', 'timestamp': '1657201806', 'distribution': '5',
+                                          'sharing_group_id': '0', 'comment': '', 'deleted': False, 'disable_correlation': False,
+                                          'first_seen': None, 'last_seen': None, 'value':
+                                          'test sender <fake_mail_sender@test.com>'},
+                                         {'id': '290725', 'event_id': '21342', 'object_id': '39958', 'object_relation':
+                                          'return-path', 'category': 'Payload delivery', 'type': 'email-src', 'value1':
+                                          '<fake_mail_sender@test.com>', 'value2': '', 'to_ids': True, 'uuid':
+                                          '87a10898-31d2-439f-b293-d268ab3e5b42', 'timestamp': '1657201806', 'distribution': '5',
+                                          'sharing_group_id': '0', 'comment': '', 'deleted': False, 'disable_correlation': False,
+                                          'first_seen': None, 'last_seen': None, 'value': '<fake_mail_sender@test.com>'},
+                                         {'id': '290726', 'event_id': '21342', 'object_id': '39958', 'object_relation':
+                                          'mime-boundary', 'category': 'Payload delivery', 'type': 'email-mime-boundary',
+                                          'value1': '00000000000049d09105dc4c2b77', 'value2': '', 'to_ids': False, 'uuid':
+                                          '2551c290-d2a9-4119-9dc5-c66562f938a9', 'timestamp': '1657201806', 'distribution': '5',
+                                          'sharing_group_id': '0', 'comment': '', 'deleted': False, 'disable_correlation': True,
+                                          'first_seen': None, 'last_seen': None, 'value': '00000000000049d09105dc4c2b77'}]}}
+    the_response.json = json_func
+    return the_response
+
+
+@pytest.mark.parametrize('expected_output', [(EXPECTED_ADD_OBJECT_OUTPUT)])
+def test_add_email_object(expected_output, mocker):
+    """
+    Given:
+    - entry & event id
+    When:
+    - Running add_email_object command.
+    Then:
+    - Ensure that the extraction of the information occured correctly.
+    """
+    from MISPV3 import add_email_object
+    import pymisp
+    event_id = 1231
+    demisto_args: dict = {'entry_id': "", 'event_id': event_id}
+    mocker.patch.object(demisto, "getFilePath", return_value={
+                        "path": "test_data/test email object.eml"
```

Comment:
lets add a couple more test files. You mentioned for example the file the playbook downloaded worked on one version while a gmail eml did not. Lets add both, and also an outlook example file